### DISCUSSION TITLE
Options for download video quality

### DISCUSSION THREAD

**Discussion by THELICHKING1** [aniyomiorg/aniyomi#352] [Q&A] [answered]
if there is an option to choose an anime download video quality(108p, 720p, 480p, etc.). Can someone please teach me how to do it.

**Comment by THELICHKING1** [comment, marked as answer]
oh I saw this answer from the "Issues". Thank you. 

"Go to browse an go to anime extension tap on which extension your using
you'll see the setting icon tap on it tap on preferred quality and when u
download it'll download in that quality"

**Reply by abderrahmanebenhalima** [reply]
yes
that's right, but sometimes (and it happened to me)
some episodes doesn't have The quality that I picked, so they download the standard quality (usually 360p)
so, it's better to select the quality of every episode before download it
